### DISCUSSION TITLE
Place of the nitro server in Techempower Web Framework Benchmarks

### DISCUSSION THREAD

**Discussion by mittci** [nuxt/nuxt#16350] [Questions] [answered]
### Environment

not relevant

### Reproduction

https://www.techempower.com/benchmarks/#section=data-r21&test=plaintext

### Describe the bug

No benchmarks, no participation to techempower https://www.techempower.com/benchmarks/#section=data-r21&test=plaintext

### Additional context

What is the place of the nitro server in the top at the start :) or at end?
Upper express or lower?

### Logs

Express suck? Or Node at all?

**Comment by mittci** [comment]
MicroSoft say tht ASP.Core is 11x faster Node.js, but in real only 2X-3X faster?
https://dusted.codes/how-fast-is-really-aspnet-core

**Comment by TheAlexLichter** [comment, marked as answer]
Feel free to check out https://github.com/fastify/benchmarks which benchmarks h3 (used by nitro under the hood) ☺️
